lib/infra/quests/in_memory/db.ex

```diff
@@ -34,8 +35,13 @@ defmodule Infra.Quests.InMemory.Db do
       nil ->
         {:error, Error.NotFound.exception(resource: :quest)}
 
-      _pid ->
-        {:ok, Projectionist.Store.get(InMemory.QuestStore, quest_id)}
+      pid ->
+        quest =
+          pid
+          |> InMemory.QuestServer.get_events()
+          |> Enum.reduce(InMemory.QuestServer.get_snapshot(pid), &Quest.project/2)
```

Comment:
The implementation of in memory quest server always has a snapshot. It just defaults to the init quest. Which is nice here so that I don't have to see if it exists before hand.